get-phing.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+wget http://www.phing.info/get/phing-latest.phar -O phing.phar
```

Comment:
This limits contributors to UNIX environments.  Using Phing's target to make an HTTP request would be more platform-independent. This requires contributors to install PEAR modules, so it might be a horse a piece.